lib/msal-browser/src/app/Configuration.ts

```diff
@@ -51,44 +37,28 @@ export type TelemetryOptions = {
  * - tokenRenewalOffsetSeconds    - sets the window of offset needed to renew the token before expiry
  * - navigateFrameWait            - sets the wait time for hidden iFrame navigation
  */
-export type SystemOptions = {
-    loggerOptions?: BrowserLoggerOptions;
+export type BrowserSystemOptions = SystemOptions & {
+    loggerOptions?: LoggerOptions;
     networkClient?: INetworkModule;
-    loadFrameTimeout?: number;
-    tokenRenewalOffsetSeconds?: number;
-    navigateFrameWait?: number;
-    telemetry?: TelemetryOptions
-};
-
-/**
- * Logger options
- * 
- * - piiLoggingEnabled          - Used to configure whether piiLogging is enabled. Defaults to false.
- * - loggerCallback             - Callback for logger that determines how message is broadcast. Defaults to console.
- */
-export type BrowserLoggerOptions = {
-    piiLoggingEnabled?: boolean,
-    loggerCallback?: ILoggerCallback
+    popupWindowTimeout?: number;
 };
 
 /**
  * Use the configuration object to configure MSAL and initialize the UserAgentApplication.
  *
  * This object allows you to configure important elements of MSAL functionality:
- * - auth: this is where you configure auth elements like clientID,  authority used for authenticating against the Microsoft Identity Platform
+ * - auth: this is where you configure auth elements like clientID, authority used for authenticating against the Microsoft Identity Platform
  * - cache: this is where you configure cache location and whether to store cache in cookies
- * - system: this is where you can configure the logger, frame timeout etc.
- * - framework: this is where you can configure the running mode of angular. More to come here soon.
+ * - system: this is where you can configure the network client, logger, token renewal offset, and telemetry
```

Comment:
I think there should be a way you can provide these comments directly on each property, so they should up in intellisense, would be worth looking into (can be done later).

Reply:
I didn't want to worry about it now, agreed we should look at this during a tech debt session.